easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.8.2-2023a.yml

```diff
@@ -18,3 +18,6 @@ easyconfigs:
       options:
         from-pr: 19268
   - X11-20230603-GCCcore-12.3.0.eb
+  - Qt5-5.15.10-GCCcore-12.3.0.eb:
+      options:
+        from-pr: 19339
```

Comment:
I think you need to explicitly add `HarfBuzz` to the easystack, and apply `from-pr` on that particular item.